fatips/102.md

```diff
@@ -14,93 +14,66 @@ tokens.
 
 Create a central, authoritative, public index of FAT tokens. Such an index is
 ideal for creating FAT token explorers, and improving the discoverability of
-tokens and the ecosystem.
+tokens within the ecosystem.
 
 
 # Specification
 
 
-## Index Chain
+## Token Index Chain
 
-A single chain, the index chain holds all entries supporting the standard
+The index chain holds entries corresponding to individual signed listings of FAT tokens by their issuers.
 
 
-### Index Chain ID `extends` [FATIP-100](100)
+### Token Index Chain ID `extends` [FATIP-100](100)
 
-Resource Identifier = `fat:index`
-
-```
-Chain ID = sha256(sha256(fat:index))
-Chain ID = efeb6db82bf222a87c1fc756e09197ed6ac7672641761a019c26530967f27744
-```
-
-
-## Index Entry
-
-Entries in the index chain represent individual FAT tokens being indexed.
-
-
-### Token ID `extends` [FATIP-100](100)
-
-A `tokenID` field in the index entry acts as a reference to the FAT token being
-indexed. This spec is compatible with FAT tokens implementing the
-[FATIP-100](100) Chain ID Derivation Standard.
+The first entry index chain has the following External IDs:
 
+| External ID Index | External ID Value |
+| ----------------- | ----------------- |
+| 0                 | "index"           |
+| 1                 | "tokens"          |
 
-### Salt
-
-A `salt` field in the index entry acts a source of randomness for the
-signature. It is a user defined string and is optional. Using a good salt helps
-ensures signatures are not re-playable
+For example, for FAT-0 token `mytoken`:
 
+```
+Chain ID = sha256(sha256("index") | sha256("tokens"))
```

Comment:
there is no mytoken used here

Reply:
Why would mytoken be needed in a chain derivation of a central index chain for all FAT tokens?

Reply:
They why does it mention mytoken above? It's a bit weird to have a variable defined, but having no dependency on it after.

Reply:
Sorry to waste your time with all these errors, will proofread better next time :+1:
The standard is meant to define a singular chain for any FAT token to be listed on by their issuers.

Reply:
This also needs a real chain ID since this is just a filler

Reply:
Resolved in commit [142f059](https://github.com/Factom-Asset-Tokens/FAT/pull/23/commits/142f059e100ed6a6e33674e6d91a108a0a45a9ea)